README.md

```diff
@@ -1 +1,134 @@
-# integration-ts-example
\ No newline at end of file
+# integration-ts-example
+
+## Introduction
+
+This guide demonstrates the basic usage of the [Node.js API wrapper for the UC Integration API](https://github.com/unfoldedcircle/integration-node-library) to develop custom drivers for [Unfolded Circle Remote Devices](https://www.unfoldedcircle.com). In this example, we will create three sample entities: a light, a media player, and a button.
+
+## Table of Contents
+1. [Prerequisites](#prerequisites)
+2. [Authentication](#authentication)
+3. [Basic Usage](#basic-usage)
+
+## Prerequisites
+
+Ensure the following are installed:
+
+- **Node.js (version 20.16.0 or higher)**: Ensure Node.js is installed on your machine. You can download it from [nodejs.org](https://nodejs.org/).
+
+- **UC Integration API**: This example includes the UC Integration API as a dependency. If you plan to develop a driver in a separate project,
+you can install the API with the following command:
+
+```
+npm install uc-integration-api
+```
+
+- **Docker Desktop**: Download [Docker Desktop](https://www.docker.com/products/docker-desktop) and install it. Start Docker, and you should see "Engine Running" in the bottom left
+corner of the window.
+
+![docker-desktop](https://github.com/user-attachments/assets/e7d8ecb6-dfc4-47f8-8a75-f0d6f7d22484)
+
+- **Remote-Core Simulator**: Download the [Remote-Core Simulator](https://github.com/unfoldedcircle/core-simulator). Navigate to `/docker`:
+
+```
+cd path/to/core-simulator/docker
+```
+
+Run:
+
+```
+docker compose-up
+```
+
+You should see messages similar to this in the terminal:
+
+![docker_compose-up](https://github.com/user-attachments/assets/5e3dc24f-dbe2-4347-bd57-ab78e23bc50b)
+
+
+Open `localhost:8080` in a browser to see the Remote Two Core Simulator:
+
+![remote-two](https://github.com/user-attachments/assets/c433a11f-7f23-4e55-963e-1f10f28808dd)
+
+Select "Web-Configurator". Enter `1234` then click "Unlock":
```

Comment:
Please don't list pin, this might change in future Simulator releases.
Change to:

    Select "Web-Configurator". Enter the pin, then click "Unlock". See [Simulator documentation](https://github.com/unfoldedcircle/core-simulator?tab=readme-ov-file#user-accounts-1) for the pin.

Reply:
I changed it.